extension/popup/ui.js

```diff
@@ -154,6 +154,15 @@ this.ui = (function() {
     });
   };
 
+  exports.showCard = function showCard(data) {
+    document.querySelector("#popup").classList.add("hasCard");
+    document.querySelector("#card-header").innerText = data.Heading;
```

Comment:
`.textContent` would be the better thing to set here (innerText is a little quirky, mostly good as a getter)

Reply:
Oops! Sorry.